manifests/kube-apiserver.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-apiserver
+  namespace: {{ .KubeAPIServerConfig.Namespace }}
+  labels:
+    tier: control-plane
+    k8s-app: kube-apiserver
+    openshift.io/control-plane: true
+    openshift.io/component: api
+  annotations:
+    openshift.io/run-level: 0
+spec:
+  selector:
+    matchLabels:
+      tier: control-plane
+      k8s-app: kube-apiserver
+  template:
+    metadata:
+      labels:
+        tier: control-plane
+        k8s-app: kube-apiserver
+      annotations:
+        checkpointer.alpha.coreos.com/checkpoint: "true"
+    spec:
+      containers:
+      - name: kube-apiserver
+        image: {{ .KubeAPIServerConfig.Image }}
+        imagePullPolicy: {{ .KubeAPIServerConfig.ImagePullPolicy }}
+        command:
+        - /usr/bin/hyperkube
```

Comment:
@deads2k @sttts this is copied from bootkube cluster up branch... I think we were in favor of having master.json (which can be empty JSON file) and just rely on the defaults @deads2k set... I need to check if the bootkube bootstrap manifest use something extra vs. the daemonset, but I guess we should remove these flags, right?